R/authenticate.R

```diff
@@ -8,15 +10,12 @@
 #'   
 #' @return sets environment variable with login information
 #' @export
-bwg_auth <- function(username, password) {
+bwg_auth <- function() {
   ## should check if it exists, and not rerun the function.
-  token_exists <- exists("token", envir = parent.frame())
-  
-  if (token_exists) stop("you've already got a token")
 
-  ## check inputs
-  assertthat::assert_that(is.character(username))
-  assertthat::assert_that(is.character(password))
+  username <- getPass::getPass("USERNAME: ")
```

Comment:
I would be inclined to use `readline("USERNAME: ")` here (with a little sanity checking) because in general the convention is username is echoed, password is not (and I :heart: conventions for code).